lib/tictactoe.rb

```diff
@@ -33,6 +37,28 @@ def begin_player_turn
 
   attr_accessor :board, :current_token, :io, :presenter
 
+  def winning_combinations
```

Comment:
I'm not sure if the tictactoe class is the best place for storing the combinations. Winning combinations depends on the board. Here in the tictactoe class we assume that we know what the board looks like.

Reply:
Thanks for the catch. I meant for winning combinations to be a list of positions a token needs to win. Because it is a win condition and board doesn't have any core game logic (like is the game over, whose turn it is) it is in `TicTacToe`.

However, I realize that I zeroed out the positions in `winning combination` and it shouldn't be since `TicTacToe` doesn't know how board is structured. I will change that. Let me know what you think about it.

Reply:
I guess the question isn't about whether we're asking the board for position 1,2,3 or 0,1,2. The fact that the tictactoe class knows to ask for 3 tokens from the board class is a sign that it's coupled to the board class. I think rather than asking the board for tokens and then find the winner, we can just ask the board who the winner is. What do you think?